src/main/java/org/openstreetmap/atlas/geography/atlas/change/FeatureChange.java

```diff
@@ -51,6 +62,18 @@ public Rectangle bounds()
         return this.reference.bounds();
     }
 
+    @Override
+    public boolean equals(final Object other)
```

Comment:
Great! I was going to add this as part of my PR. Glad you already did. We will need one for the `Change` class as well.